rllib/algorithms/marwil/marwil.py

```diff
@@ -178,7 +178,7 @@ def build(
     ) -> "Algorithm":
         if not self._set_off_policy_estimation_methods:
             deprecation_warning(
-                old="MARWIL used to have off_policy_estimation_methods "
+                old=r"MARWIL used to have off_policy_estimation_methods "
                 "is and wis by default. This has"
                 "changed to off_policy_estimation_methods: \{\}."
```

Comment:
When we move to an r string, do we have to remove the escapes here?
```suggestion
                "changed to off_policy_estimation_methods: {}."
```

Reply:
Good eye. This PR passes tests as-is, but the next PR (where I enable failure-on-warning except for known issues) fails on this. I fix it [here](https://github.com/ray-project/ray/pull/31219/files#diff-e26bca85df649219577cf98e8a56da338494252c81a6c3859d7625b852fece95R183).

I'm not sure why it isn't a problem for this PR but is a problem for the next one. I don't think it's an issue since the tests pass.

Reply:
To answer your question, only the first string literal is converted to a raw string literal. See this example:
```python
print('No raw strings literals')
print("1 2 \n3 4 5"
      " 6 7 \n8 9 10")

print('First string literal is a raw string literal')
print(r"1 2 \n3 4 5"
      " 6 7 \n8 9 10")

print('Both string literals are raw string literals')
print(r"1 2 \n3 4 5"
      r" 6 7 \n8 9 10")
```
```
No raw strings literals
1 2
3 4 5 6 7
8 9 10
First string literal is a raw string literal
1 2 \n3 4 5 6 7
8 9 10
Both string literals are raw string literals
1 2 \n3 4 5 6 7 \n8 9 10
```

Reply:
Oh I see! thanks for the explanation